v1/branch/branch.gunk

```diff
@@ -139,39 +138,45 @@ type UpdateBranchRequest struct {
 	// Metadata is the branch metadata.
 	Metadata string `pb:"7" json:"metadata"`
 }
+
 // DeleteBranchRequest is the request envelope to delete a branch.
-type DeleteBranchRequest struct{
+type DeleteBranchRequest struct {
 	// BranchID is the branch unique identifier.
 	BranchID string `pb:"1" json:"branch_id"`
 }
 
-// GetBranchRequest is a request to get the branch data.
+// GetBranchRequest is the request envelope to get the branch data.
 type GetBranchRequest struct {
 	// BranchID is the branch unique identifier.
 	BranchID string `pb:"1" json:"branch_id"`
 }
 
 // GetBranchesResponse is the response for GetBranches
-type GetBranchesResponse struct{
+type GetBranchesResponse struct {
 	// Result is the list of the branch.
 	Result []Branch `pb:"1" json:"result"`
 }
 
-// BranchService provides branch operations.
+// BranchService provides branch-related operations.
 type BranchService interface {
-	// GetBranch retrieves the detail of a branch information, selected by its ID.
+	// GetBranch retrieves the details of a specific branch information, selected by its ID.
 	//
 	// +gunk http.Match{
 	//         Method: "GET",
 	//         Path:   "/v1/branches/{BranchID}",
 	// }
 	// +gunk openapiv2.Operation{
 	//         Tags:        []string{"Branch"},
-	//         Description: "Retrieves branch information, selected by the BranchID you supplied.",
+	//         Description: "Retrieve information about the branch specified by the ID",
 	//         Summary:     "Retrieve branch information",
 	//         Responses: map[string]openapiv2.Response{
 	//                 "200": openapiv2.Response{
 	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/branchBranch",
```

Comment:
Should the Ref be "#/definitions/branch`es`Branch"?

Reply:
similar to above: https://github.com/openbank/openbank/blob/b496bf680a51b141c09f9cb480b133d4d54aba6e/v1/branch/all.swagger.json#L294